stdlib/websub/src/main/ballerina/src/websub/natives.bal

```diff
@@ -69,7 +67,7 @@ function addSubscription(SubscriptionDetails subscriptionDetails) = @java:Method
 #
 # + topic - The topic for which the update should happen
 # + content - The content to send to subscribers, with the payload and content-type specified
-# + return - `error` if an error occurred during publishing
+# + return - An `error` if an error occurred during publishing or esle `()`
```

Comment:
```suggestion
# + return - An `error` if an error occurred during publishing or esle `()`
```
Need to specify the error.

Reply:
We are using a common error in this module. So we don't need to specify the error.